command/src/state.rs

```diff
@@ -63,6 +63,22 @@ impl ConfigState {
     }
   }
 
+  pub fn contains_backend(&self, app_id: &str, address: &SocketAddr) -> bool {
```

Comment:
those modifications to `ConfigState` are not needed

Reply:
Why ?

Reply:
because they are not used at all in this pull request

Reply:
Oh yes ! now I don't need it anymore ... good catch :smile: